forbiddenapis/signatures.txt

```diff
@@ -0,0 +1,3 @@
+com.puppycrawl.tools.checkstyle.DefaultConfiguration#getAttributeNames() @ Usage of deprecated API is forbidden. Please use DefaultConfiguration.getPropertyNames() instead.
```

Comment:
Please move this file to "config" folder

Reply:
done